test_e2e.sh

```diff
@@ -310,6 +310,42 @@ assert_file_eq "$ROOT"/link/file "$TESTCASE 1"
 # Wrap up
 pass
 
+##############################################
+# Test worktree-cleanup
+##############################################
+testcase "worktree-cleanup"
+
+echo "$TESTCASE" > "$REPO"/file
+git -C "$REPO" commit -qam "$TESTCASE"
+GIT_SYNC \
+    --wait=20 \
```

Comment:
Make --wait=0.1 to reduce the wait time between syncs

Reply:
We want enough time to add the second commit and create a worktree. So 10s atleast ?